Feature/icon minicart (#236)
vtex-apps/minicart: docs/README.md

| `openOnHover`          | `boolean`                 | Whether the `popup` minicart should open when the user hovers over it.                                                                                                                                                                       | `false`         |
| `quantityDisplay`      | `enum`                    | Shows the quantity badge even when the amount of products in the cart is zero. Possible values are: `always` or `not-empty` or `never`).                                                                                                                | `not-empty` |
| `itemCountMode` | `enum` | Quantity badge behavior when displaying the number of total items added in Minicart. Possible values are: `total`  (quantity badge displays the number of items added to the cart) or `distinct` (quantity badge only displays the number of different products added to the cart). | `distinct` |

> | `backdropMode`         | `enum` | Controls whether the backdrop should be displayed when the `drawer` Minicart is opened or not. Possible values are: `visible` (rendering the backdrop) or `none` (rendering the `drawer` without backdrop).  | `none` |
> | `MinicartIcon` | `block` | The icon block that should be rendered inside the minicart button | `vtex.store-icons@0.x:icon-cart` |

Reviewer: ```suggestion
| `backdropMode`         | `enum` | Controls whether the backdrop should be displayed when the `drawer` Minicart is opened or not. Possible values are: `visible` (rendering the backdrop) or `none` (rendering the `drawer` without backdrop).  | `none` |
| `MinicartIcon` | `block` | Icon displayed in the Minicart button. This prop's value must match the name of the block responsible for rendering the desired icon. | `icon-cart` (from [Store Icons](https://vtex.io/docs/components/all/vtex.store-icons/) app) |
```

```jsonc
// This is the default blocks implementation for the minicart-layout
{
  "icon-expand#minicart-icon": {
    "props": {
      "size": 24
    }

>   },

Reviewer 2: I would rather move this bellow the minicart definition.

I think the top-down method is clearer. Bigger thing > to smaller things.

Author: I always do the opposite haha (define first and then use it). But I think your approach makes more sense

Reviewer 2: More on this from Clean Code:
https://dzone.com/articles/the-stepdown-rule

// This is the default blocks implementation for the minicart-layout
{
  "minicart.v2": {
    "props": {

>       "MinicartIcon": "icon-expand#minicart-icon"

Reviewer 3: ```suggestion
      "MinicartIcon": "icon-cart#minicart-icon"
```

    },
    "children": ["minicart-base-content"]
  },

>   "icon-expand#minicart-icon": {

Reviewer 3: ```suggestion
  "icon-cart#minicart-icon": {
```
